modules/sandboxed-containers-enable-debug-crio.adoc

```diff
@@ -6,68 +6,36 @@
 [id="sandboxed-containers-enable-debug-logs_{context}"]
 = Enabling debug logs for {sandboxed-containers-first}
 
-As a cluster administrator, you can collect a more detailed level of logs for {sandboxed-containers-first}. Enhance logging by changing the `log_level` in the CRI-O runtime for the worker nodes running {sandboxed-containers-first}.
+As a cluster administrator, you can collect a more detailed level of logs for {sandboxed-containers-first}. Enhance logging by changing the `logLevel` field in the KataConfig CR. This changes the `log_level` in the CRI-O runtime for the worker nodes running {sandboxed-containers-first}.
```

Comment:
I see `KataConfig` in monospace in the docs. I think this would need to be as well. https://docs.openshift.com/container-platform/4.11/sandboxed_containers/uninstalling-sandboxed-containers.html#sandboxed-containers-deleting-kataconfig-web-console_uninstalling-sandboxed-containers